pipeline/inputs/process-exporter-metrics.md

```diff
@@ -0,0 +1,103 @@
+
+## Metrics Available
+
+| Name              | Description                                                                                      |
+| ----------------- | -------------------------------------------------- |
+| cpu               | Exposes CPU statistics from `/proc`.               |
+| io                | Exposes I/O statistics from `/proc`.               |
+| memory            | Exposes memory statistics from `/proc`.            |
+| state             | Exposes process state statistics from `/proc`.     |
+| context\_switches | Exposes context\_switches statistics from `/proc`. |
+| fd                | Exposes file descriptors statistics from `/proc`.  |
+| start\_time       | Exposes start\_time statistics from `/proc`.       |
+| thread\_wchan     | Exposes thread\_wchan from `/proc`.                |
+| thread            | Exposes thread statistics from `/proc`.            |
+
+## Getting Started
+
+### Simple Configuration File
+
+In the following configuration file, the input plugin _process\_exporter\_metrics collects _metrics every 2 seconds and exposes them through our [Prometheus Exporter](../outputs/prometheus-exporter.md) output plugin on HTTP/TCP port 2021.
+
+```
+# Process Exporter Metrics + Prometheus Exporter
+# -------------------------------------------
+# The following example collect host metrics on Linux and expose
+# them through a Prometheus HTTP end-point.
+#
+# After starting the service try it with:
+#
+# $ curl http://127.0.0.1:2021/metrics
+#
+[SERVICE]
+    flush           1
+    log_level       info
+
+[INPUT]
+    name            process_exporter_metrics
+    tag             process_metrics
+    scrape_interval 2
+
+[OUTPUT]
+    name            prometheus_exporter
+    match           process_metrics
+    host            0.0.0.0
+    port            2021
+```
+
+You can see the metrics by using _curl:_
+
+```bash
+curl http://127.0.0.1:2021/metrics
+```
+
+### Container to Collect Host Metrics
+
+When deploying Fluent Bit in a container you will need to specify additional settings to ensure that Fluent Bit has access to the process details.
+The following `docker` command deploys Fluent Bit with a specific mount path for `procfs` and settings enabled to ensure that Fluent Bit can collect from the host. 
+These are then exposed over port 2021.
+
+```
+docker run -ti -v /proc:/host/proc:ro \
+               -p 2021:2021        \
+               fluent/fluent-bit:2.2 \
+               /fluent-bit/bin/fluent-bit \
+                         -i process_exporter_metrics -p path.procfs=/host/proc  \
+                         -o prometheus_exporter -p "add_label=host $HOSTNAME" \
```

Comment:
we need to specify the `HOSTNAME` variable btw (`-e HOSTNAME="$HOSTNAME"`) or just remove it might be easier

Reply:
Sure. I removed it from the doc.